render/fx_renderer/fx_pass.c

```diff
@@ -345,6 +346,11 @@ void fx_render_pass_add_texture(struct fx_gles_render_pass *pass,
 		|| fx_options->discard_transparent;
 	setup_blending(!has_alpha ? WLR_RENDER_BLEND_MODE_NONE : options->blend_mode);
 
+	const struct wlr_box clipped_region_box = fx_options->clipped_region.area;
+	enum corner_location clipped_region_corners = fx_options->clipped_region.corners;
+	int clipped_region_corner_radius = clipped_region_corners != CORNER_LOCATION_NONE ?
+		fx_options->clipped_region.corner_radius : 0;
+
```

Comment:
I'd like to optimize things a bit by subtracting the clipped region box from the damaged area like we do with rect and shadow